src/amio-webchat.client.js

```diff
@@ -65,14 +64,10 @@ class AmioWebchatClient {
       this.socket.on(SOCKET_MESSAGE_SERVER, data => {
         this.socket.emit(SOCKET_MESSAGE_DELIVERED, {
           message_id: data.id
-        })
+        }, () => {})
```

Comment:
Is it necessary that we always return something?

Reply:
It is necessary for socket.io to play nicely on the server side. If we don't specify the callback here, then it will be undefined on the server, too.